conf/openshift/openshift.json

```diff
@@ -1,449 +1,483 @@
 {
```

Comment:
Is this file completely different because of whitespace changes? I can't tell what changed. What changed? Do you want to put the whitespace back how it was so I can answer this question myself?

Reply:
I added the initContainer object and auto format this.

Reply:
@thaorell even after you autoformatted it, every line has been changed. It's very difficult to see what you really changed.

Reply:
@thaorell in d5b1534 I documented how to use `jq` to keep the pretty printing of openshift.json consistent. Feedback is welcome. I also went ahead and changed the file back to the format it had before (2 spaces instead of 4 for indenting).